### ISSUE TITLE
Fail to deploy any workload in PSP restricted cluster

### ISSUE THREAD

**Issue by gitlawr** [rancher/rancher#22539]
**Rancher Version**
master 8/30 57508ebbaa08309b46b7766c76fff659c972e0a5

**Steps to reproduce (least amount of steps as possible):**
1. start rancher server by docker
2. Add a single node EC2 cluster with `Pod Security Policy Support: enabled, Default PSP: restricted`, everything else the default(v1.15.3-rancher3-1)
3. After the cluster is active, edit the Default project, change PSP to unrestricted.
4. Deploy a busybox deployment in the Default project.

**Result:**
![image](https://user-images.githubusercontent.com/5697937/63996038-118e0e00-cb2d-11e9-83d9-caec9d315465.png)

**Other details that may be helpful:**
Error logs in rancher server:
```
2019/08/30 05:41:11 [ERROR] PodSecurityPolicyTemplateProjectBindingController c-p675b/psptpb-bx8x4 [PodSecurityPolicyTemplateProjectBindingsLifecycleHandler] failed with : error creating policy: the API version in the data (extensions/v1beta1) does not match the expected API version (policy/v1beta1)
2019/08/30 05:41:11 [ERROR] PodSecurityPolicyTemplateProjectBindingController c-p675b/psptpb-bx8x4 [PodSecurityPolicyTemplateProjectBindingsLifecycleHandler] failed with : error creating policy: the API version in the data (extensions/v1beta1) does not match the expected API version (policy/v1beta1)
```

**Comment by cjellick** [comment]
looks related to v1.16 change @dramich

**Comment by jiaqiluo** [comment]
The bug is reproduced in master-head 57508ebba

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:master-head e50e71498

Following the same steps as the issue describes, now we can deploy workloads successfully.
